release.py

```diff
@@ -391,8 +391,10 @@ def constant_replace(
     os.rename(fn + ".new", fn)
 
 
-def tweak_patchlevel(tag: Tag, done: bool = False) -> None:
-    print("Updating Include/patchlevel.h...", end=" ")
+def tweak_patchlevel(
+    tag: Tag, fn: str = "Include/patchlevel.h", done: bool = False
```

Comment:
Can we use a more descriptive parameter name? I read `fn` as "function" at first and was confused.

Reply:
Sure, I copied `fn` from elsewhere in the file, so updated them all to `filename`.